frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -74,7 +76,51 @@ const styles = {
 };
 
 class EditEmail extends Component {
+  static propTypes = {
+    onChange: PropTypes.func.isRequired
+  };
+
+  state = {
+    emailType: EMAIL_TYPE.reply,
+    emailTo: "",
+    emailCc: "",
+    emailBody: "",
+    reasonsForAction: ""
+  };
+
+  onEmailTypeChange = event => {
+    const newEmailType = event.target.value;
+    this.setState({ emailType: newEmailType });
+    this.props.onChange({ ...this.state, emailType: newEmailType });
+  };
+
+  onemailToChange = event => {
```

Comment:
The name of this function should be with a capital 'E' (see '_email_' word) to be consistent: _on**E**mailTypeChange_

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/180/commits/a30fbec8f08be4778d32f2cc176d05761bbb495b and commit #https://github.com/code-for-canada/project-thundercat/pull/180/commits/ff90c3754c23b30b4ff2fbfcca6ef2cb0277391a